apitools/base/py/credentials_lib.py

```diff
@@ -384,6 +387,21 @@ def _refresh(self, _):
     self.access_token = token
 
 
+def _GetRunFlowFlags():
+  if FLAGS is None:
```

Comment:
yes -- because of the way `gflags` works, the `auth_local_webserver` flag only gets declared if the `tools` module was loaded. so at some point i had an internal build where `FLAGS` was available, but `FLAGS.auth_local_webserver` threw an `AttributeError`.

i found and fixed the root cause in that case, but it seemed safe enough to leave the code as extra cautious, in the event it ever came back.